app/views/users/index.html.erb

```diff
@@ -2,15 +2,16 @@
 <html>
 <head>
   <title>All Users</title>
+  <link rel="stylesheet" type="text/css" href="/assets/stylesheets/users.css">
 </head>
 <body>
   <h1>All Users</h1>
-  <ul>
+  <ul class="user-list">
     <% @users.each do |user| %>
-      <li>
+      <li class="user-item">
         <img src="<%= user.photo %>" alt="<%= user.name %>">
         <h1><%= user.name %></h1>
-        <p>Number of Posts: <%= user.posts.count %></p>
+        <p class="user-info">Number of Posts: <%= user.posts.count %></p>
       </li>
     <% end %>
   </ul>
```

Comment:
- [ ] The users on the index page should have links that take the user to each user's show page. On the layout page, you can see there is space between the picture and the user details. And the number of posts part is towards the right of the container.

- [ ] Kindly ensure your design follows [the layout](https://raw.githubusercontent.com/microverseinc/curriculum-rails/main/blog-app/images/blog_root_users.png?token=GHSAT0AAAAAACG3ADWRWSADJVFDEMCUEZDWZHM4BRQ) for the user index page

  ![image](https://github.com/mpa-mxiang/Blog-app/assets/81468675/903dd8df-d279-4397-a2f6-d6e60be72ca5)

Reply:
Hi @julie-ify , it shows me `404: Not Found` when I open the link, could you please update it?